docs/intro.md

```diff
@@ -3,4 +3,31 @@ title: User Manuals
 sidebar_position: 1
 ---
 
-WIP
+## Introduction
+
+Pando is a decentrialized financial network built with [Mixin Trusted Group](https://developers.mixin.one/document/mainnet/mtg), includes a series of open financial protocols.
+
+Being based on a trusted network mechanism, Pando combines most common decentralized financial applications into one product to serve users and discover new fininancial applications.
+
+Pando integrates the following financial protocols, all of them can be run stand-alone.
+
+### Pando Lake & 4swap
+
+**[4swap](https://4swap.org)** is a decentralized protocol implement for automated liquidity provision on Mixin Network, which is a community-driven project that allow for any pool to be added.
+
+4swap provides a mechnism named "Broker" that users interact with the protocol. [**Pando Lake**](lake/intro)  has own Broker to serve the users. In additional, **Pando Lake** has a plan to provide advanced fininancial products based on 4swap and other Pando series products.
```

Comment:
mechnism --> **mechanism**

Pando Lake has **its** own Broker...

In additional --> **Additionally,**